api/src/server.js

```diff
@@ -33,5 +33,17 @@ app.use('/publications', publicationsRouter);
 app.use('/team', teamRouter);
 app.use('/theme', themeRouter);
 
+// error handler middleware
+app.use(function (err, req, res) {
+  const errorObject = err;
+  if (errorObject) {
+    res.status(err.errorCode).json(err);
+  } else {
+    // if error object is not passed, server error
+    res.status(500).send('Something went wrong!');
```

Comment:
Do we need to change this to something else other than 'Something went wrong'?

Reply:
Yes we do, thanks for the reminder!